source/features/repo-age.tsx

```diff
@@ -17,6 +16,27 @@ const dateFormatter = new Intl.DateTimeFormat('en-US', {
 	day: 'numeric'
 });
 
+const getCommitCount = async (): Promise<AnyObject> => {
```

Comment:
No need to extract this into its own function though, especially considering the AnyObject typing. Just merge it back into getFirstCommit

Reply:
Besides that everything else is ok?

Reply:
I remember why I split it.  I was having issues with constants they both are returning the same names.

Reply:
I don't know what you're talking about

Reply:
That's what happens when I reply half in my sleep. Sorry I misunderstood you I thought you wanted both api requests in one function.

Anyways I'm up now ☕